frontend/src/components/teacher/session-creation/steps/ChooseClass.tsx

```diff
@@ -34,8 +34,8 @@ const ChooseClass = ({
     <DistributeAssessmentWrapper
       emptyState={<EmptyClassroomsGoToPageMessage />}
       isEmpty={data?.length === 0}
-      isError={Boolean(error)}
-      isLoading={Boolean(loading)}
+      isError={error}
```

Comment:
Does this work now? I remember this cast being necessary at one point

Reply:
yea tested that it works. not sure why it was needed for loading (and error now takes in ApolloError not boolean)